app_dart/lib/src/service/build_status_provider.dart

```diff
@@ -114,9 +115,9 @@ class BuildStatusService {
   ///
   /// The returned stream will be ordered by most recent commit first, then
   /// the next newest, and so on.
-  Stream<CommitStatus> retrieveCommitStatus({int limit, int timestamp, String branch}) async* {
+  Stream<CommitStatus> retrieveCommitStatus({int limit, int timestamp, String branch, String repo}) async* {
```

Comment:
Make this `RepositorySlug` to enforce the format

Reply:
done.